### PR TITLE
Add standalone configuration

### PR DESCRIPTION
**Pull request by lgalis**
https://issues.redhat.com/browse/SSP-2492

Add standalone configuration for approval UI.
Configuration only in this PR. The Navigation sidebar and the login component will be federated from the catalog in a future PR.
Screenshot from 2021-11-05 14-18-51

### CODE REVIEW

File: config/standalone.dev.webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const webpackBase = require('./webpack.base.config');
+
+// Used for getting the correct host when running in a container
+const proxyHost = process.env.API_PROXY_HOST || 'localhost';
+const proxyPort = process.env.API_PROXY_PORT || '5001';
+const apiBasePath = process.env.API_BASE_PATH || '/api/ansible-catalog/v1';
```
Comment (eclarizio) [inline config/standalone.dev.webpack.config.js:6]: Are we going to need to change these to something less generic? For example for the API, all of the environment variables are going to be prefixed with `ANSIBLE_CATALOG_`, should the same be true for these? We probably need to change the ones in catalog-ui, too.

**Review by eclarizio — COMMENTED** [review]
One concern about the environment variable naming, since we were discussing it about the API code as well. Could be pushed off into a separate PR, but would like to hear your thoughts.

File: config/standalone.dev.webpack.config.js (hunk 1 above)
Comment (lgalis) [reply config/standalone.dev.webpack.config.js:6]: @eclarizio - agree - we should change the names to use the same prefix

File: config/webpack-ts-overrides.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { resolve } = require('path');
+
+const isBuild = process.env.NODE_ENV === 'production';
```
[the remark this answers is not in the document]
Comment (lgalis) [reply config/webpack-ts-overrides.js:4]: This needs to stay as is, it is used by node as well.

**Review by eclarizio — APPROVED** [review]

**Pull request merged**